### ISSUE TITLE
Partner Theme to be replaced with Areas of Focus as filter on Projects page

### ISSUE THREAD

**Issue by PeaceGeeks** [PeaceGeeksSociety/PeaceGeeks-Website#76]
With the night to reflect on it, introducing 'Partner Theme' does not make sense. (really sorry for the change). Instead of using a taxonomy fields it should be referercning of the the 5 'Areas of Focus'. I will delete Partner Theme once this change is made.

**Comment by reallyordinary** [comment]
Why does Partner Theme not make sense? 

You want to swap that out with an entityreference to Areas of Focus and be able to filter by the titles of the Areas of Focus nodes?

**Comment by PeaceGeeks** [comment]
That is correct. Sorry about the change - I am really conscious of not pulling stuff like this. 

Because we are introducing a new idea, when we have one that already kind of overlaps with some of these through broader themes - the themes I included in that list are essentially all sub-themes of 'Areas of Focus'. At some point, we would like to be abel to click on an Area of Focus and show all of the projects. This seems like the more logical and clean way to manage the data.

**Comment by reallyordinary** [comment]
Then shouldn't you change "Partner Theme" to something like "Areas of Focus Theme/Type" and attach that taxonomy to the Areas of Focus content type?

**Comment by reallyordinary** [comment]
In addition to attaching Areas of Focus entityreference to the Partner content type and removing the taxonomy from Partner.

**Comment by reallyordinary** [comment]
This needs a lot more work in order to get the Areas of Focus entityreference field to work as a filter on the Projects page. Its default is an autocomplete text field. No easy way to turn it into dropdown or checkboxes/radio buttons. I suggest leaving this until post launch because it's going to potentially take... well... awhile.
